searchlib/src/vespa/searchlib/features/distancetopathfeature.cpp

```diff
@@ -132,17 +131,16 @@ DistanceToPathBlueprint::createExecutor(const search::fef::IQueryEnvironment &en
     std::vector<Vector2> path;
     search::fef::Property pro = env.getProperties().lookup(getName(), "path");
     if (pro.found()) {
-        vespalib::string str = pro.getAt(0);
+        vespalib::stringref str = pro.getAt(0);
         uint32_t len = str.size();
         if (str[0] == '(' && len > 1 && str[len - 1] == ')') {
             str = str.substr(1, len - 1); // remove braces
-            std::vector<vespalib::string> arr;
-            boost::split(arr, str, boost::is_any_of(","));
-            len = arr.size() - 1;
+            vespalib::StringTokenizer tokenizer(str);
+            len = tokenizer.size() - 1;
```

Comment:
Can the contents of the `()` ever be empty/just whitespace? `vespalib::StringTokenizer` implicitly prunes away a token list consisting of a single empty token, so it seems that an input of `str == "()"` would cause `tokenizer.size() == 0` which in turn would cause an integer underflow. Consider checking for empty tokens case, or `assert` if it should never happen.

Reply:
Fixed in #25883